src/components/Cataloguing/Marc/Record/index.js

```diff
@@ -336,11 +361,11 @@ class Record extends React.Component<
 
   deleteRecord = async () => {
     let { statusCode } = this.state;
-    const { recordDetail, store, router, reset } = this.props;
+    const { recordDetail, store, router, reset, translate, toggleFilterPane, data: { search: { segment } } } = this.props;
     await del(
       buildUrl(
         store.getState(),
-        C.ENDPOINT.BIBLIOGRAPHIC_RECORD + '/' + recordDetail.id,
+        segment === C.SEARCH_SEGMENT.BIBLIOGRAPHIC ? C.ENDPOINT.BIBLIOGRAPHIC_RECORD + '/' + recordDetail.id : C.ENDPOINT.AUTHORITY_RECORD + '/' + recordDetail.id,
```

Comment:
I recommend using string interpolation here to make this easier to read.

Reply:
ok, I have changed the code using string interpolation